src/Umbraco.Web.BackOffice/Trees/TreeCollectionBuilder.cs

```diff
@@ -58,15 +58,33 @@ public void AddTreeControllers(IEnumerable<Type> controllerTypes)
                 AddTreeController(controllerType);
         }
 
-        public void RemoveTreeController<T>() => RemoveTreeController(typeof(T));
+        public void RemoveTree(Tree treeDefinition)
+        {
+            if (treeDefinition == null)
+                throw new ArgumentNullException(nameof(treeDefinition));
+            _trees.Remove(treeDefinition);
+        }
+
+        public void RemoveTreeController<TController>()
+            where TController : TreeControllerBase
+            => RemoveTreeController(typeof(TController));
 
-        public void RemoveTreeController(Type type)
+        public void RemoveTreeController(Type controllerType)
```

Comment:
Unfortunately we can't do this, if anyone has used `RemoveTreeController(type: MyType)` then their code will break. 😅

Reply:
Yeah, it seems you are right. Maybe that should be fixed in v10 to make the arguments consistent? Maybe other part of the code where this isn't consistent?

![image](https://user-images.githubusercontent.com/2919859/155307298-47370c38-6c7a-44e3-a01d-bb4a500d20b6.png)

Reply:
I have reverted the change in parameter name. Maybe @bergmania @p-m-j or @nikolajlauridsen knows if we should fix this in v10?

Reply:
That indeed seems like a reasonable thing to fix in v10 to me 👍 I mean it's not the end of the world but it would be nice to be consistent 😄

Reply:
@nul800sebastiaan I think this should be good now and we could eventually fix it in v10.